woof-code/rootfs-packages/imageview/usr/sbin/imgview

```diff
@@ -30,10 +30,16 @@ MANUAL=true
 [ "$SLIDE" == 'true' ] && MANUAL=false
 
 cd "$CWD"
-
-read DIMS SCRX SCRY ROOTVERT << EOF
-`xwininfo -root | grep ' \-geometry ' | cut -f 1 -d '+' | tr 'x' ' '`
+if [ "$XDG_SESSION_TYPE" = 'wayland' ]; then
```

Comment:
Why XDG_SESSION_TYPE and not WAYLAND_DISPLAY? Who sets this one?

Reply:
I'll change that.

Reply:
Done